packages/common/src/scorekeeper/jobs/types.ts

```diff
@@ -0,0 +1,44 @@
+import { ApiHandler, ChainData, Config, Constraints } from "../../index";
+import MatrixBot from "../../matrix";
+import Nominator from "../../nominator/nominator";
+
+export type JobRunnerMetadata = {
+  config: Config.ConfigSchema;
+  chaindata: ChainData;
+  nominatorGroups: Nominator[];
+  nominating: boolean;
+  bot: MatrixBot;
+  constraints: Constraints.OTV;
+  handler: ApiHandler;
+  currentTargets: { stash?: string; identity?: any }[];
+};
+
+export type JobConfig = {
+  jobKey: keyof Config.ConfigSchema["cron"] | "";
+  defaultFrequency: string;
+  jobFunction: (metadata: JobRunnerMetadata) => Promise<any>;
+  name: string;
+  preventOverlap?: boolean;
+};
+
+// There is a dependency on status names in scorekeeper-status-ui
+type StatusName =
+  | "running"
+  | "finished"
+  | "errored"
+  | "started"
+  | "Not Running";
```

Comment:
why `Not Running` starts with capital letters ? 
They should probably be constants/enum defined somewhere rather than hardcoded strings, wdyt ?

Reply:
100% agree, that's the plan.
I commented, `There is a dependency on status names in scorekeeper-status-ui`. I decided not to touch it just to minimize the impact of the current PR. Also, I need to check if more dependencies on this status name exist.